site/src/sphinx/advanced-structured-logging.rst

```diff
@@ -148,6 +152,107 @@ automatically for you:
         }
     }
 
+Enabling content previews
+-------------------------
+Armeria provides the ``requestContentPreview`` and ``responseContentPreview`` properties in :api:`RequestLog`
+to retrieve the textual representation of the first N bytes of the request and response content.
+However, the properties are disabled by default due to performance overhead and thus they always return ``null``.
+You can enable it when you configure :api:`Server`, :api:`VirtualHost` or :api:`Client`.
+
+.. code-block:: java
+
+    import com.linecorp.armeria.server.ServerBuilder;
+    import com.linecorp.armeria.server.VirtualHostBuilder;
+
+    ServerBuilder sb = new ServerBuilder();
+    ...
+    // Enable previewing the content with the maxium length of 100 for textual content.
+    sb.contentPreview(100);
+    ...
+    VirtualHostBuilder vhb = new VirtualHostBuilder("http://example.com");
+    // In this case, the property of virtual host takes precedence over that of server.
+    vhb.contentPreview(150);
+    ...
+    sb.virtualHost(vhb.build());
+
+.. code-block:: java
+
+    import com.linecorp.armeria.client.HttpClientBuilder;
+
+    HttpClientBuilder cb = new HttpClientBuilder();
+    ...
+    cb.contentPreview(100);
+
+Note that the ``contentPreview()`` method enables the previews only for textual content
+which meets the following cases:
```

Comment:
nit: meets the following cases -> meets one of the following cases? (to make more clear)